ONSdigital/eq-author-app
# 1435 enabling commenting for survey introduction and section introduction pages

### What is the context of this PR?
Here we are adding commenting/Replies to the following pages:
1. Introduction page
2. Section Pages
3. Confirmation Pages

In order for the resolvers to handle returns - Each query now needs a key/value pair instead of just an Id (eg: introductionID: "id") so the resolver can identify the type of page we are dealing with and return the relevant results since the data structure is different when dealing with these different types of pages.

### How to review 
Create a new BUSINESS survey.
Commenting has been added to the PREVIEW part of each of the pages. so you will probably need to add a page title in order to activate the preview tab (next to Design tab)

On each of the pages above:
 you will see a comments panel in the right hand side.
Please test that you can add/delete/edit **comments** and the same for **replies**.

To see a confirmation page - hit "add" on a question page.

To test that **subscriptions** are working - 
Open a chrome incognito instance of Author and log in with a different user - 
At the same time as logged in on different user on another chrome tab. (on another screen)
When updating comments/replies - you should automatically see the other instance of author refreshing (as long as you are on the same page) - 
so multiple users logged into the same page will be able to send comments in real time!
